marlowe-apps/src/Language/Marlowe/Runtime/App/Parser.hs

```diff
@@ -101,14 +93,26 @@ chainSeekCommandPort :: CliOption O.OptionFields PortNumber
 chainSeekCommandPort = port "chain-seek-command" "CHAINSEEK_COMMAND" 3720 "The port number of the chain-seek server's job API."
 
 
-chainSeekQueryPort :: CliOption O.OptionFields PortNumber
-chainSeekQueryPort = port "chain-seek-query" "CHAINSEEK_QUERY" 3716 "The port number of the chain-seek server's query API."
-
-
 chainSeekSyncPort :: CliOption O.OptionFields PortNumber
 chainSeekSyncPort = port "chain-seek-sync" "CHAINSEEK_SYNC" 3715 "The port number of the chain-seek server's synchronization API."
 
 
+syncHost :: CliOption O.OptionFields HostName
+syncHost = host "marlowe-sync" "SYNC" "127.0.0.1" "The hostname of the Marlowe Runtime marlowe-sync server."
+
+
+syncSyncPort :: CliOption O.OptionFields PortNumber
+syncSyncPort = port "marlowe-sync-sync" "SYNC_SYNC" 3724 "The port number of the marlowe-sync server's synchronization API."
+
+
+syncHeaderPort :: CliOption O.OptionFields PortNumber
+syncHeaderPort = port "marlowe-sync-header" "SYNC_HEADER" 3725 "The port number of the marlowe-sync server's header synchronization API."
+
+
+syncQueryPort :: CliOption O.OptionFields PortNumber
+syncQueryPort = port "marlowe-sync-query" "SYNC_QUERY" 3726 "The port number of the marlowe-sync server's query API."
+
+
```

Comment:
Could you please move these definitions to `Language.Marlowe.Runtime.CLI.Option` so other places can use them?

Reply:
Fixed in dd7cbd5b7646c2d03f5ee88274dab8653a21aba8.